### PR TITLE
Add env default processor

### PR DESCRIPTION
**Pull request by jderusse**
This PR documents symfony/symfony#28976

### CODE REVIEW

**Comment by jderusse** [comment]
PR symfony/symfony#29409 merged the code sample should be ok

File: configuration/external_parameters.rst [hunk 1]
```diff
@@ -494,6 +494,46 @@ Symfony provides the following env var processors:
...
+    Retrieves the value of the parameter ``fallback_param`` when the of the ``BAR`` env var is not available:
+
+    .. configuration-block::
+
+        .. code-block:: yaml
+
+            # config/services.yaml
+            parameters:
+                private_key: '%env(default:raw_key:file:PRIVATE_KEY)%'
+                raw_key: '%env(PRIVATE_KEY)%'
```
Comment (javiereguiluz) [inline configuration/external_parameters.rst]: I wonder if we could put a hardcoded path/value for `raw_key` to make the example easier to understand. Using an env var as the default value of the env var value of another value makes my head explode 💥

File: configuration/external_parameters.rst [hunk 2]
```diff
@@ -494,6 +494,46 @@ Symfony provides the following env var processors:
     .. versionadded:: 4.2
         The ``key`` processor was introduced in Symfony 4.2.
 
+``env(default:fallback_param:BAR)``
+    Retrieves the value of the parameter ``fallback_param`` when the of the ``BAR`` env var is not available:
```
Comment (ogizanagi) [inline configuration/external_parameters.rst]: This sentence seems to be missing some or having extra words :)

```diff
- Retrieves the value of the parameter ``fallback_param`` when the of the ``BAR`` env var is not available:
+ Retrieves the value of the parameter ``fallback_param`` when the ``BAR`` env var is not available:
```

**Comment by javiereguiluz** [comment]
@jderusse thanks for contributing this! I merged it and did the fixes suggested by Maxime.

**Pull request merged**